### ISSUE TITLE
two URIs for Sergius of Tella

### ISSUE THREAD

**Issue by dlschwartz** [srophe/srophe-app-data#704]
I'm pretty sure that person/1463 and person/2072 are the same person. Both are the Sergius of Tella who became patriarch of Antioch. This is recounted in work/839 (Patrologia Orientalis vol. 18.4 (1924), : p.689 [487]). Our record for work/839 indicates that it commemorates person/2072.

**Comment by nathangibson** [comment]
OK, looks like we should merge person/1463 and person/2072 and correct the PO citation in person/1463 which should be to 18.4, not 17.1. Correct?

**Comment by dlschwartz** [comment]
Yes, the merge and the change in PO volume are both correct. Thank you Nathan! Please let me know when you decide which URI to deprecate and which to keep. Thanks.

**Comment by nathangibson** [comment]
We will keep person/1463.

**Comment by dlschwartz** [comment]
Great thanks!

**Comment by nathangibson** [comment]
Done, and I've assigned person/1463 to you in Perseids (we could use it for the testing on Tu.)

**Comment by dlschwartz** [comment]
Yes, let's do these on Tuesday. Thanks!

**Comment by davidamichelson** [comment]
@wsalesky @dlschwartz @nathangibson Something is not right here, this record person/2072 still displays even though it was maybe deprecated?

**Comment by wlpotter** [comment]
@jsaintlauren we should take a look at this one in conjunction with #668
